Week3/homework/step2-3.js

```diff
@@ -30,10 +33,10 @@ console.log('while', repeatStringNumTimesWithWhile('abc', 3));
 function repeatStringNumTimesWithDoWhile(str, num) {
   // eslint-disable-next-line
   let result = '';
-
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
-
+  do {
```

Comment:
If `num <=0` the loop body of the `do-while` loop will be executed, where it shouldn't. This is because with a `do-while` loop the loop condition is tested at the end of the loop, making `do-while` rather useless in practice. You can always rewrite a `do-while` as a regular `while` loop. But here, where you are asked to use a `do-while` loop you must add an `if` statement to check whether the `do-while` loop should be executed at all.